src/service/eventService.ts

```diff
@@ -0,0 +1,93 @@
+import { Request, Response, NextFunction } from 'express';
+import { eventRepository } from '../repository';
+import { logger } from '../config/winston';
+import { fail, success } from '../modules/util';
+import { responseMessage, statusCode } from '../modules/constants';
+import EventDto from '../dto/event/eventDto';
+
+const getEvent = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const events = await eventRepository.getAllEvents();
+
+    if (!events) {
+      logger.info(`Events를 찾을 수 없습니다.`);
+      return fail(res, statusCode.NOT_FOUND, responseMessage.NOT_FOUND);
+    }
+    return success(res, statusCode.OK, responseMessage.SUCCESS, events);
+  } catch (error) {
+    next(error);
+  }
+};
+
+const getPopup = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const { recentId } = req.params;
+    const event = await eventRepository.getInProgressEvent();
+    if (!event) {
+      logger.info(`Events를 찾을 수 없습니다.`);
+      return fail(res, statusCode.NOT_FOUND, responseMessage.NOT_FOUND);
+    }
+    if (+recentId == event.id) {
+      return success(
+        res,
+        statusCode.FORBIDDEN,
+        responseMessage.DONT_SHOW_POPUP,
+      );
+    }
+    return success(res, statusCode.OK, responseMessage.SUCCESS, event);
+  } catch (error) {
+    next(error);
+  }
+};
+
+const makeEvent = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const { title, type, target, eventStatus } = req.body;
+    const eventDto: EventDto = {
+      title,
+      type,
+      target,
+      eventStatus,
+    };
```

Comment:
```suggestion
    const eventDto: EventDto = req.body;
```

바로 req.body에서 가져오면 코드는 좀 더 질 것 같습니다!
이렇게 했을때 발생되는 다른 문제가 있을까요??

Reply:
이게 더 깔끔하네요! 미처 알지 못했던 방법입니다. 반영했습니다!

Reply:
> 그리고 계속 synk이 PR을 올리는데 그렇게 중요한 것 같지도 않고 PR만 혼잡하게 되는거 같아서 Synk PR 기능만 빼면 어떨까요?

좋은생각입니다!